src/Raven.Studio/typescript/components/pages/database/tasks/ongoingTasks/editTasks/editGenAiTask/partials/fields/EditGenAiTaskModelFields.tsx

```diff
@@ -62,28 +62,39 @@ export default function EditGenAiTaskModelFields() {
                 <div>
                     <div className="mb-1">
                         JSON schema
-                        <PopoverWithHoverWrapper message="TODO">
+                        <PopoverWithHoverWrapper
+                            message={
+                                <>
+                                    Enter a formal JSON schema that defines the structure of the response you want to
+                                    receive from the model.
```

Comment:
Looking at this text again... pls modify to:
`Enter a JSON schema that defines the structure of the output you expect from the model.`